src/main.js

```diff
@@ -10,6 +11,14 @@ Vue.use(vueDebounce, {
   listenTo: 'input'
 });
 
+if (process.env.SENTRY_DSN_ENDPOINT){
+  Sentry.init({
+    Vue: Vue,
+    dsn: process.env.SENTRY_DSN_ENDPOINT,
+    logErrors: true 
```

Comment:
add the environment so that we can specify which environment is being reported the error